permission_handler_platform_interface/pubspec.yaml

```
@@ -3,13 +3,13 @@ description: A common platform interface for the permission_handler plugin.
 homepage: https://github.com/baseflow/flutter-permission-handler/tree/master/permission_handler_platform_interface
 # NOTE: We strongly prefer non-breaking changes, even at the expense of a
 # less-clean API. See https://flutter.dev/go/platform-interface-breaking-changes
-version: 2.0.1
+version: 3.0.0-nullsafety.0
```

Please remove the trailing `.0` as we are following the pattern of not adding a `.0` for the first `nullsafety` release.
```suggestion
version: 3.0.0-nullsafety
```